client/src/frontend/containers/Home/Collections.js

```diff
@@ -12,17 +12,47 @@ export default function HomeCollectionsContainer() {
     []
   );
 
-  const { data } = useFetch({
-    request: [projectCollectionsAPI.index, filters]
+  const journalFilters = useMemo(
+    () => ({
+      showOnHomepage: true
+    }),
+    []
+  );
+
+  const { data: collections } = useFetch({
+    request: [projectCollectionsAPI.index, filters],
+    refetchOnAuthChange: true
   });
 
-  if (!data) return null;
-  return data.map((projectCollection, index) => (
-    <EntityCollection.ProjectCollectionSummary
-      key={projectCollection.id}
-      projectCollection={projectCollection}
-      limit={projectCollection.attributes.homepageCount}
-      bgColor={index % 2 === 0 ? "neutral05" : "white"}
-    />
-  ));
+  const { data: journals } = useFetch({
+    request: [journalsAPI.index, journalFilters],
+    refetchOnAuthChange: true
+  });
+
+  const journalCount = journals?.length;
+
+  if (!collections && !journals) return null;
+
+  return (
+    <>
+      {journals &&
+        journals.map((journal, i) => (
+          <EntityCollection.JournalIssues
+            key={journal.id}
+            journal={journal}
+            bgColor={i % 2 === 0 ? "neutral05" : "white"}
+            limit={8}
+          />
+        ))}
+      {collections &&
+        collections.map((projectCollection, i) => (
+          <EntityCollection.ProjectCollectionSummary
+            key={projectCollection.id}
+            projectCollection={projectCollection}
+            limit={projectCollection.attributes.homepageCount}
+            bgColor={(journalCount + i) % 2 === 0 ? "neutral05" : "white"}
+          />
+        ))}
+    </>
+  );
```

Comment:
I see this comment in the design:
> For now, publishers can choose to have their featured Journals appear at the top of the home page, or at he bottom (i.e. below the Projects)

@zdavis sounds like this needs to be added to the API somewhere. Or do we want to just fix the order for now?

Reply:
For now, let's just put them at the top.